super_dpf.py

```diff
@@ -222,6 +225,56 @@ def sync(self, verify_local):
         if not cloud_filelist:
             logging.info('{} has nothing to sync.'.format(self.service_name))
 
+        logging.info('Done syncing {}'.format(self.service_name))
+
+
+class UnsplashResource(BaseDPF):
+    REQUIRED_SETTINGS = [
+        SettingsItem('preferred_res', 'Preferred Resolution',
+                     help_text='eg 1600x900'),
+        SettingsItem('category', 'Unsplash Category',
+                     help_text='choices: buildings, food, nature, people,'
+                               ' technology, objects, none', primary=True)
+    ]
+
+    @property
+    def service_name(self):
+        return 'Unsplash'
+
+    def sync(self, verify_local):
+        logging.info('Syncing {}'.format(self.service_name))
+        full_path = '{}/{}'.format(PATHS['photos'], self.subdir)
+        duplicates = list()
+
+        if self.settings.get('category') == 'none' or False:
+            url = '{}/{}/{}'.format(UNSPLASH['base'], UNSPLASH['random'],
+                                    self.settings.get('preferred_res'))
+        else:
+            url = '{}/{}/{}/{}'.format(UNSPLASH['base'], UNSPLASH['category'],
+                                       self.settings.get('category'),
+                                       self.settings.get('preferred_res'))
+
+        response = requests.get(url)
+        img_hash = hashlib.md5(response.request.path_url).hexdigest()
+        filename = '{}/{}.jpg'.format(full_path, img_hash)
+        if not os.path.isfile(filename):
+            logging.info('Getting img at {}'.format(response.url))
+            try:
+                img = Image.open(StringIO(response.content))
+                img.save('{}'.format(filename), 'JPEG', optimize=True,
+                         progressive=True, quality=100, subsampling=0)
+            except IOError as e:
+                logging.error(e)
+        else:
+            duplicates.append(filename)
+
+        if duplicates:
+            logging.info(
+                'Skipped {} duplicate images from {}'.format(
+                    len(duplicates), self.service_name))
+
+        logging.info('Done syncing {}'.format(self.service_name))
```

Comment:
I find it preferable to initialize a logger using logging.getLogger('name') rather than use the logging.<level> functions directly.  I have had odd problems when using logging.<leve>() functions.